app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
```

Comment:
The function should check if the command starts with 'mv' as specified in the checklist: 'Make sure you check that the input string contains 3 elements separated by space, and has `mv` on a first place.'